anthos-bm-gcp-terraform/terraform.tfvars.sample

```diff
@@ -10,7 +10,8 @@ instance_count                = {
                                   "controlplane" : 3
                                   "worker" : 2
                                 }
-#gce_vm_service_account       = "<GCE service account with iam.serviceAccountKeyAdmin permission>"
+#gcp_login_accounts           = "<GCP_ACCOUNT_EMAIL_TO_USE_FOR_CLUSTER_LOGIN>"
```

Comment:
super duper nit:
i see in the `yaml` that you can input a list for the emails  (`gcpAccounts: [foo@gmail.com, bar@gmail.com, bazz@google.com]`) Im guessing that it's also a list here?  maybe a commented example might be helpful?

Reply:
Makes sense and good catch. Added a list example placeholder for this